### PR TITLE
Switch to copier template

### PR DESCRIPTION
**Pull request by znicholls**
- [x] entry added in `changelog`

### CODE REVIEW

**Comment by maritsandstad** [comment]
I can try to clean up the ciceroscm pandas< 2 issue @znicholls, if that's helpful (probably about time anyway)...

**Comment by znicholls** [comment]
Would be great, I'll ping you once I've stopped moving things here so you
have something stable to work from

**Comment by maritsandstad** [comment]
Great, I'll wait for a ping, then :-)

**Comment by znicholls** [comment]
@lewisjared not really sure if review can do anything meaningful but just as an FYI. Docs build based on notebooks and have runtime in there, which is kind of cool (e.g. https://openscm-runner--85.org.readthedocs.build/en/85/notebooks.html#notebook-execution-info). 

Basically all worked. Had to do a bunch of fixes but not that surprising given how complicated the repo is. I'm going to leave mypy failing for now (too much legacy to fix that right now). I'll take the learnings back into the copier repo next week.

File: pyproject.toml [hunk 1]
```diff
@@ -1,34 +1,260 @@
...
+    "Development Status :: 4 - Beta",
+    "License :: OSI Approved :: BSD License",
+    "Intended Audience :: Developers",
+    "Operating System :: OS Independent",
+    "Intended Audience :: Science/Research",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
```
Comment (lewisjared) [inline pyproject.toml:40]:
```suggestion
    "Programming Language :: Python :: 3.9",
```

File: pyproject.toml [hunk 2]
```diff
@@ -1,34 +1,260 @@
...
+python = "^3.9"
+openscm-units = ">=0.5.0"
+python-dotenv = "^1.0.0"
+scmdata = ">=0.7.4"
+tqdm = "^4.66.1"
+# optional below here
+ciceroscm = { version = "1.0.0", optional = true }  # TODO: ask Marit if she wants to loosen this
+fair = { version = "<2", optional = true }
+pymagicc = { version = "<3", optional = true }
+# Bit hacky, but basically we only need to pin pandas
```
Comment (lewisjared) [inline pyproject.toml:56]: Is this because `ciceroscm` v1.0.0 doesn't pin pandas <2?

**Review by lewisjared — APPROVED** [review]
Nice work grinding through this

File: pyproject.toml (hunk 2 above)
Comment (znicholls) [reply pyproject.toml:56]: Yep, Marit is going to follow up once I'm finished here

**Pull request merged**